### ISSUE TITLE
Question: Does bleve support index sorting?

### ISSUE THREAD

**Issue by darren** [blevesearch/bleve#1265]
Elasticsearch Introduced Index Sorting in 6.0:

https://www.elastic.co/cn/blog/index-sorting-elasticsearch-6-0

So does bleve support this similar feature?

**Comment by will7200** [comment]
Yes It does as defined [here](https://blevesearch.com/docs/Sorting/).
> Simple API
>
>The simple API adds a SortBy() method to the SearchRequest. Here is an example which sorts by the >field age.
>
>searchRequest.SortBy([]string{"age"})

**Comment by darren** [comment]
@will7200 

No, that's not Index sorting as in ElasticSearch. 

The `Index Sort` is  to sort documents on disk at index time, not at search time. it is very useful for query time series data.

`searchRequest.SortBy([]string{"age"})` is sorting query results at search time.

**Comment by will7200** [comment]
@darren

You are right, I should’ve read the link and not make assumptions.

**Comment by mschoch** [comment]
Currently we do not support this.  I will convert this question to track the feature request.

**Comment by 4kord** [comment]
+1 for the feature, it would be nice to have index sorting
